### PR TITLE
Fix SBDB query handling of responses with single epoch

### PR DESCRIPTION
**Pull request by spenczar**
Lots of test infrastructure too

### CODE REVIEW

File: adam_core/orbits/query/tests/test_sbdb.py [hunk 1]
```diff
@@ -41,3 +47,37 @@ def test__convert_SBDB_covariances():
         )
 
     npt.assert_equal(cometary_covariances, adam_core_format)
+
+
+def test_query_sbdb_for_ceres():
+    with mock_sbdb_query("Ceres.json") as mock:
+        result = query_sbdb(["Ceres"])
+        mock.assert_called_once()
+
```
Comment (spenczar) [inline adam_core/orbits/query/tests/test_sbdb.py:56]: Maybe could add more significant tests here.

File: adam_core/orbits/query/sbdb.py [hunk 2]
```diff
@@ -184,3 +197,12 @@ def query_sbdb(ids: npt.ArrayLike) -> Orbits:
     classes = np.array(classes)
 
     return Orbits(coordinates, object_ids=object_ids)
+
+
+class NotFoundError(Exception):
```
Comment (moeyensj) [inline adam_core/orbits/query/sbdb.py:202]: I'm a fan of this addition.

File: adam_core/orbits/query/sbdb.py [hunk 3]
```diff
@@ -133,38 +137,47 @@ def query_sbdb(ids: npt.ArrayLike) -> Orbits:
...
+        orbit = result["orbit"]
+        elements = orbit["elements"]
+        epoch = orbit["epoch"]
+        if "covariance" in orbit:
+            covariances_sbdb[i, :, :] = orbit["covariance"]["data"]
+
+            if "elements" in orbit["covariance"]:
+                # If elements is provided inside covariance, then it's
+                # the elements at the epoch which was used to
+                # calculate covariance, so we should prefer it.
```
Comment (moeyensj) [inline adam_core/orbits/query/sbdb.py:155]: Nice, thanks for the clear comment

**Review by moeyensj — APPROVED** [review]
Great! Code was clear to read and easy to understand, and the tests are a cherry.

**Pull request merged**